src/main/java/core/basesyntax/service/impl/DataParserServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataParserServiceImpl implements DataParserService {
+    private static final String LINE_SPLITTER = ",";
+    private static final byte OPERATION_INDEX = 0;
+    private static final byte FRUIT_INDEX = 1;
+    private static final byte QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseDataToFruitTransaction(List<String> data) {
+        return data.stream()
+            .skip(1)
```

Comment:
It's magic number. Move it to constant field.

Reply:
we need to make constants of data, which can be changed or not to be obvious. like indexes or separators. it's just skipping the first line

Reply:
ok